modules/nw-dns-loglevel.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+// * networking/dns/nw-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-dns-loglevel_{context}"]
+= Set CoreDNS logLevel
+
+Valid values for CoreDNS logLevel are `Normal`, `Debug`, and `Trace`.
```

Comment:
You can configure CoreDNS `logLevel` to determines the amount of detail in logged error messages. Valid values for CoreDNS logLevel are `Normal`, `Debug`, and `Trace`.

[NOTE]
====
The errors plugin is always enabled. The following `logLevel` settings will report different error responses: 

- `logLevel`: `Normal` will enable the "errors" class: `log . { class error }`.

- `logLevel`: `Debug` will enable the "denial" class: `log . { class denial error }`.

- `logLevel`: `Trace` will enable the "all" class: `log . { class all }`.
====